docs/seqcol.md

```diff
@@ -364,6 +366,7 @@ inherent:
   - sequences
 ```
 
+
 #### 3.2 Collection
 
 - *Endpoint*: `GET /collection/:digest?level=:level` (`REQUIRED`)
```

Comment:
With the addition of an `attribute` endpoint, I think the `collection` endpoint should now move down to `RECOMMENDED`

Reply:
after discussion we decided to leave both required for now.